pkg/pipelines/bootstrap.go

```diff
@@ -5,16 +5,30 @@ import (
 	"fmt"
 	"os"
 	"path"
+	"strings"
 
 	"github.com/mitchellh/go-homedir"
 	"github.com/openshift/odo/pkg/pipelines/eventlisteners"
 	"github.com/openshift/odo/pkg/pipelines/routes"
 	"sigs.k8s.io/yaml"
 )
 
+// DefaultTokenFileName is the default token file name
+var DefaultTokenFileName string = "~/Downloads/token/token.txt"
```

Comment:
This isn't the same as in the bootstrap script, it uses `token`.

But, I think this is probably nicer, we just need to document it somewhere.

Reply:
I guess during squashing, the commits got messed up. I had kept  it as "~/Downloads/token"